database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,39 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // $this->call(UsersTableSeeder::class);
+        DB::table('users')->insert([
```

Comment:
Same here, you can use Eloquent models to query the database. Same goes for all the seed file